diff --git a/node_modules/beatsaver-api/lib/api/search.js b/node_modules/beatsaver-api/lib/api/search.js
index 2914394..19f32be 100644
--- a/node_modules/beatsaver-api/lib/api/search.js
+++ b/node_modules/beatsaver-api/lib/api/search.js
@@ -13,8 +13,6 @@ var SortOrder;
 function searchMaps(axiosInstance, searchOptions = { sortOrder: SortOrder.Latest }, page = 0) {
     return (0, tslib_1.__awaiter)(this, void 0, void 0, function* () {
         try {
-            if (searchOptions.q)
-                searchOptions.q = encodeURIComponent(searchOptions.q);
             const response = yield axiosInstance.get(`/search/text/${page}`, { params: searchOptions });
             return response.data;
         }
